e2e/abort_scan_test.go

```diff
@@ -0,0 +1,77 @@
+// Copyright © 2023 Cisco Systems, Inc. and its affiliates.
+// All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package e2e
+
+import (
+	"fmt"
+	"github.com/onsi/ginkgo/v2"
+	"github.com/onsi/gomega"
+	"github.com/openclarity/vmclarity/api/models"
+	"github.com/openclarity/vmclarity/pkg/shared/utils"
+	"time"
```

Comment:
Does the linters skip the e2e folder? not sure how goimport linter was happy here :)

Reply:
We definitely need to fix this.

Reply:
https://github.com/golangci/golangci-lint/issues/828, @chrisgacsal will fix this in a separate pull request

Reply:
Opened a PR with the fix for this issue: https://github.com/openclarity/vmclarity/pull/596